### ISSUE TITLE
not-authorized, Registering status ok

### ISSUE THREAD

**Issue by olebowle** [davidgfnet/whatsapp-purple#295]
Hi,

since today I'm getting not-authorized when I try connecting to WhatsApp:
```
(10:50:07) dnsquery: Performing DNS lookup for c2.whatsapp.net
(10:50:07) dns: Successfully sent DNS request to child 5177
(10:50:07) dns: Got response for 'c2.whatsapp.net'
(10:50:07) dnsquery: IP resolved for c2.whatsapp.net
(10:50:07) proxy: Attempting connection to 173.192.222.169
(10:50:07) proxy: Connecting to c2.whatsapp.net:443 with no proxy
(10:50:07) proxy: Connection in progress
(10:50:07) proxy: Connecting to c2.whatsapp.net:443.
(10:50:07) proxy: Connected to c2.whatsapp.net:443.
(10:50:09) connection: Connection error (reason: 2 description: not-authorized)
(10:50:09) jabber: jabber_actions: have pep: YES
```
I'm using the latest git (0.8.3.4.g2817d0c-1) with the server "c2.whatsapp.net" and resource "Android-2.12.5-443". I tried registering with sms once more and got a new password with expiry date of 28. Sep 2015, "type":"existing", "kind":"free" and "status":"ok". So I don't seem to be blocked by WhatsApp. I get the same error while connecting with the new password. Did they change their API somehow?

Cheers,
olebowle

**Comment by JayFoxRox** [comment]
Same problem since yesterday.
Re-registering also OK, new password didn't help.

Same resource + server, german number (+49).
Happened after traveling from my home wifi to my parents wifi so it might be related to that?

**Comment by davidgfnet** [comment]
Yep, confirmed myself. Change the "resource" to be "Android-2.12.176-443" instead of the original "Android-2.12.5-443" works great.

**Comment by wopaum** [comment]
I am not receiving messages with this resource x-X

**Comment by wopaum** [comment]
can't receive messages with this resource

**Comment by davidgfnet** [comment]
We can try S40 too:
S40-14.26-443

Does that work?

**Comment by NeoBelerophon** [comment]
I can confirm that "S40-14.26-443" is working as well

**Comment by erwin68** [comment]
Got this "not-authorized" too, when I started pidgin today.
But login with yowsup worked fine.
Yesterday evening, after installing, pidgin was working too.
Then I only changed resource to Android-2.31.151-443 and pidgin worked again.

**Comment by Adeor** [comment]
that fixed it for me too ... thx ^^
any idea why this happened?

**Comment by NeoBelerophon** [comment]
Obviously whatsapp disallows the old client version

**Comment by davidgfnet** [comment]
WART was updated today, so I guess it's related:
https://github.com/mgp25/WART/

**Comment by olebowle** [comment]
Fixed with 75d5f56, thanks!
